/**:
  ros__parameters:
    debug: false
    vehicle_ip: "192.168.2.2"
    timeout: 15
    mission_filename: "Misione_1.kml"
    vehicle_id: 3
    mqtt_addr: "171.0.0.1"
    pump: false
    num_samples: 4
    time_between_samples: 0.5
    travelling_timeout: 60
    arm_timeout: 15
    max_distance: 5000
    connect_timeout: 15
    simple_goto_timeout: 10
    internet_loss_timeout: 30
    sensor_read_timeout: 15
    override_control: false
